pkg/ranger/file.go

```diff
@@ -8,10 +8,10 @@ import (
 	"os"
 )
 
-// FileHandleRanger returns a RangerCloser from a file handle. The
-// RangerCloser's Close method will call fh.Close().
+// FileHandleRanger returns a Closer from a file handle. The
+// Closer's Close method will call fh.Close().
 // Footgun: If FileHandleRanger fails, fh.Close will not have been called.
-func FileHandleRanger(fh *os.File) (RangerCloser, error) {
+func FileHandleRanger(fh *os.File) (Closer, error) {
```

Comment:
I think changing `RangerCloser` to `Closer` here and in all other places is wrong.

Reply:
It fixes the linter. The issue being when you import the package you will call ranger.RangerCloser which stutters. instead it should user ranger.Closer

Reply:
So linter complains because the type name starts with the package name?

How about if you rename it to `RangeCloser` (note the missing `r` in the middle) instead of `Closer`. Similar to `Reader` -> `ReadCloser`, we should have `Ranger` -> `RangeCloser`.

Reply:
That should be good. The thought is if it is a "ranger closer" then ranger.closer get you there. I'll change to rangeCloser